web/src/beta/components/fields/DateTimeField/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+import Property from "..";
+import TextInput from "../common/TextInput";
+
+export type Props = {
+  name?: string;
+  description?: string;
+  value?: string;
+  onChange?: (value?: string | undefined) => void;
+};
+
+const DateTimeField: React.FC<Props> = ({ name, description, value, onChange }) => {
+  const [time, setTime] = useState<string>();
+  const [date, setDate] = useState<string>();
+
+  useEffect(() => {
+    if (value) {
+      const [dateString, timeString] = value.split(" ");
+      setTime(timeString);
+      setDate(dateString);
+    }
+  }, [value]);
+
+  useEffect(() => {
+    if (value) onChange?.(date + " " + time);
+  }, [date, onChange, time, value]);
+
+  return (
+    <Property name={name} description={description}>
+      <Wrapper>
```

Comment:
The `Property` component already is flexing, so I don't think you need this Wrapper component 🙏🏻

Reply:
No it flex as rows , I need one row here